RELEASE_NOTES.md

```diff
@@ -1,20 +1,54 @@
-# Cosmos SDK v0.43.0-RC3 Release Notes
+# Cosmos SDK v0.43.0 Release Notes
 
-This is a fourth Release Candidate for v0.43. If no issue will be reported this will be the final release.
+This release introduces several new important updates to the Cosmos SDK. The release notes below provide an overview of the larger high-level changes introduced in the v0.43 release series.
 
-This release contains two major bug fixes:
+That being said, this release does contain many more minor and module-level changes besides those mentioned below. For a comprehsive list of all breaking changes and improvements since the v0.42 "Stargate" release series, please see the [CHANGELOG](https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-rc0/CHANGELOG.md).
```

Comment:
```suggestion
That being said, this release does contain many more minor and module-level changes besides those mentioned below. For a comprehsive list of all breaking changes and improvements since the v0.42 "Stargate" release series, please see the [CHANGELOG](https://github.com/cosmos/cosmos-sdk/blob/v0.43.0/CHANGELOG.md).
```